src/Enum.php

```diff
@@ -35,21 +37,22 @@ abstract class Enum implements JsonSerializable
 
     /**
      * @return string[]
-     * @psalm-return array<string|int, string>
+     * @psalm-return array<TEnumValue, string>
      */
```

Comment:
Question, do we still need the differentiation between psalm and other array types?
Looks like PHPStan supports the `array<key,value>` syntax, so removing the `@return` and dropping the `psalm` prefix would benefit?

Would that be okay?

https://phpstan.org/writing-php-code/phpdoc-types#general-arrays

Reply:
I'm not sure about PHPStorm and other IDEs or any other doc-comment parsers.
I would target this in a different issue/PR.

Reply:
Can confirm on PhpStorm, but aye. Baby steps 'n all